x/storage/keeper/keeper.go

```diff
@@ -512,25 +532,25 @@ func (k Keeper) CreateObject(
 	store.Set(types.GetBucketByIDKey(bucketInfo.Id), bbz)
 
 	obz := k.cdc.MustMarshal(&objectInfo)
-	store.Set(objectKey, sequence.EncodeSequence(objectInfo.Id))
+	store.Set(objectKey, k.objectSeq.EncodeSequence(objectInfo.Id))
 	store.Set(types.GetObjectByIDKey(objectInfo.Id), obz)
 
 	if err := ctx.EventManager().EmitTypedEvents(&types.EventCreateObject{
-		Creator:          operator.String(),
-		Owner:            objectInfo.Owner,
-		BucketName:       bucketInfo.BucketName,
-		ObjectName:       objectInfo.ObjectName,
-		BucketId:         bucketInfo.Id,
-		ObjectId:         objectInfo.Id,
-		CreateAt:         objectInfo.CreateAt,
-		PayloadSize:      objectInfo.PayloadSize,
-		Visibility:       objectInfo.Visibility,
-		PrimarySpAddress: bucketInfo.PrimarySpAddress,
-		ContentType:      objectInfo.ContentType,
-		Status:           objectInfo.ObjectStatus,
-		RedundancyType:   objectInfo.RedundancyType,
-		SourceType:       objectInfo.SourceType,
-		Checksums:        objectInfo.Checksums,
+		Creator:        operator.String(),
+		Owner:          objectInfo.Owner,
+		BucketName:     bucketInfo.BucketName,
+		ObjectName:     objectInfo.ObjectName,
```

Comment:
We may need to expose GVG or LVG id for offchain services.

Reply:
Also for SealObject message

Reply:
When the object is created, the information about gvg or lvg is not known. It is only available when sealing object . The related information has been added to the EventSealObject.

Reply:
if it is empty object, then it will get bindings, i think.